src/server.ts

```diff
@@ -5,6 +5,7 @@ import { ApiManager } from "./chain_service/api.js";
 import Faucet from "./chain_service/faucet.js";
 import { AddressError, RateError } from "./chain_service/faucetErrors.js";
 import { SlackNotifier } from "./slack_service/slack.js";
+import { handler } from "../interface/build/handler.js";
```

Comment:
I took the whole idea of importing handler.js here from the official Svelte documentation: https://kit.svelte.dev/docs/adapter-node#custom-server

If we want our svelte application to be served via Express - we can do it with handler.js imported from frontend build. I think importing from the build is not a problem, if we configure Netlify correctly to create the build frontend first, then handlers.js will always exist

Reply:
An obvious downside of this approach is that we cannot use the backend without the frontend anymore. 

We probably wouldn't want to do this anyway, but can we maybe configure this somehow so that the express server only adds the handler if the file `handler.js` exists, and if it doesn't, we ignore it?